config/samples/istio_v1beta1_istio_cni_gke.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: istio.banzaicloud.io/v1beta1
+kind: Istio
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: istio-sample
+spec:
+  mtls: false
+  includeIPRanges: "*"
+  excludeIPRanges: ""
+  autoInjectionNamespaces:
+  - "default"
+  controlPlaneSecurityEnabled: false
+  sds:
+    enabled: false
+  pilot:
+    image: "docker.io/istio/pilot:1.1.0-rc.4"
+    replicaCount: 1
+    minReplicas: 1
+    maxReplicas: 5
+    traceSampling: 1.0
+  citadel:
+    image: "docker.io/istio/citadel:1.1.0-rc.4"
+    replicaCount: 1
+  galley:
+    image: "docker.io/istio/galley:1.1.0-rc.4"
+    replicaCount: 1
+  gateways:
+    ingress:
+      replicaCount: 1
+      minReplicas: 1
+      maxReplicas: 5
+      serviceAnnotations: {}
+      serviceLabels: {}
+    egress:
+      replicaCount: 1
+      minReplicas: 1
+      maxReplicas: 5
+      serviceAnnotations: {}
+      serviceLabels: {}
+    k8singress:
+      enabled: false
+  mixer:
+    image: "docker.io/istio/mixer:1.1.0-rc.4"
+    replicaCount: 1
+    minReplicas: 1
+    maxReplicas: 5
+  sidecarInjector:
+    image: "docker.io/istio/sidecar_injector:1.1.0-rc.4"
+    replicaCount: 1
+    rewriteAppHTTPProbe: true
+    initCNIConfiguration:
+      enabled: true
+      binDir: /home/kubernetes/bin
```

Comment:
The only difference between `istio_v1beta1_istio_cni.yaml` and `istio_v1beta1_istio_cni_gke.yaml` is this line: `binDir: /home/kubernetes/bin`. The rest of the yaml is the same all the way. I'm wondering whether we should use some kind of templating for these samples to get rid of duplicated lines and make the samples easier to maintain. For example, do you think using `kustomize` would be too much here?